tests/fixture/testnet/config.go

```diff
@@ -278,17 +284,16 @@ func (nc *NodeConfig) EnsureNodeID() error {
 	return nil
 }
 
-type AddrAndBalance struct {
-	Addr    ids.ShortID
-	Balance uint64
-}
+// Helper type to simplify configuring X-Chain genesis balances
+type XChainBalanceMap map[ids.ShortID]uint64
```

Comment:
I feel like just using `map[ids.ShortID]uint64` instead of defining a type for it would suffice